packages/blade/src/components/Card/__tests__/CardInteractive.web.test.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { CardBody, Card } from '../';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+import { Box } from '~components/Box';
+import { VisuallyHidden } from '~components/VisuallyHidden/VisuallyHidden';
+import { Text } from '~components/Typography';
+
+describe('<Card />', () => {
+  it('should render a interactive Card', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}} scaleOnHover isSelected={true}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a clickable Card', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card', () => {
+    const { container } = renderWithTheme(
+      <Card href="https://google.com">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card with onClick', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}} href="https://google.com">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should click on the card', async () => {
+    const user = userEvent.setup();
+    const onClick = jest.fn();
+    const { getByRole } = renderWithTheme(
+      <Card onClick={onClick} accessibilityLabel="My Card">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    await user.click(getByRole('button', { name: /My Card/i }));
+    expect(onClick).toHaveBeenCalled();
+  });
+
+  it('should behave properly when buttons are present', async () => {
+    const user = userEvent.setup();
+    const rootOnClick = jest.fn();
+    const childOnClick = jest.fn();
+    const { getByRole } = renderWithTheme(
+      <Card onClick={rootOnClick} accessibilityLabel="My Card">
+        <CardBody>
+          <button onClick={childOnClick}>Child Button</button>
+        </CardBody>
+      </Card>,
+    );
+
+    await user.tab();
+    expect(getByRole('button', { name: /My Card/i })).toHaveFocus();
+    await user.click(getByRole('button', { name: /My Card/i }));
+    expect(rootOnClick).toHaveBeenCalled();
+
+    await user.tab();
+    expect(getByRole('button', { name: /Child Button/i })).toHaveFocus();
+    await user.click(getByRole('button', { name: /Child Button/i }));
+    expect(childOnClick).toHaveBeenCalled();
```

Comment:
```suggestion
    expect(childOnClick).toHaveBeenCalled();
     expect(rootOnClick).toNotHaveBeenCalled();
```

Reply:
Changed it to use `toHaveBeenCalledTimes(1)`